template-bff-service/webpack.config.js

```diff
@@ -23,10 +24,11 @@ module.exports = {
     minimize: false
   },
   target: 'node',
-  mode: slsw.lib.webpack.isLocal ? "development" : "production",
+  mode: slsw.lib.webpack.isLocal ? 'development' : 'production',
   externals: [
     nodeExternals()
   ],
+  plugins: includeMocks() ? [new EnvironmentPlugin({ REPLAY: process.env.REPLAY })] : undefined,
```

Comment:
allow developers to set the REPLAY env var from the cli when running/recording tests